File: relationship-manager/notification-service/src/main/java/com/bank/rm/notification/NotificationServiceApplication.java
package com.bank.rm.notification;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.notification", "com.bank.rm.common"})
public class NotificationServiceApplication {
    public static void main(String[] args) { SpringApplication.run(NotificationServiceApplication.class, args); }
}

File: relationship-manager/notification-service/src/main/kotlin/com/bank/rm/notification/NotificationServiceApplication.kt
package com.bank.rm.notification

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.notification", "com.bank.rm.common"])
class NotificationServiceApplication

fun main(args: Array<String>) {
    runApplication<NotificationServiceApplication>(*args)
}
